src/components/AttachmentPicker/index.native.js

```diff
@@ -318,65 +242,118 @@ class AttachmentPicker extends Component {
-        return this.props.children({
-            openPicker: ({onPicked}) => this.open(onPicked),
+    function renderChildren() {
+        return children({
+            openPicker: ({onPicked}) => open(onPicked),
         });
     }
 
-    render() {
-        return (
-            <>
-                <Popover
-                    onClose={this.close}
-                    isVisible={this.state.isVisible}
-                    anchorPosition={styles.createMenuPosition}
-                    onModalHide={this.onModalHide}
-                >
-                    <View style={this.props.isSmallScreenWidth ? {} : styles.createMenuContainer}>
-                        <ArrowKeyFocusManager
-                            focusedIndex={this.state.focusedIndex}
-                            maxIndex={this.menuItemData.length - 1}
-                            onFocusedIndexChanged={(index) => this.setState({focusedIndex: index})}
-                        >
-                            {_.map(this.menuItemData, (item, menuIndex) => (
-                                <MenuItem
-                                    key={item.textTranslationKey}
-                                    icon={item.icon}
-                                    title={this.props.translate(item.textTranslationKey)}
-                                    onPress={() => this.selectItem(item)}
-                                    focused={this.state.focusedIndex === menuIndex}
-                                />
-                            ))}
-                        </ArrowKeyFocusManager>
-                    </View>
-                </Popover>
-                {this.renderChildren()}
-            </>
-        );
-    }
+    return (
+        <>
+            <Popover
+                onClose={() => close()}
+                isVisible={isVisible}
+                anchorPosition={styles.createMenuPosition}
+                onModalHide={onModalHide.current}
+            >
+                <View style={isSmallScreenWidth ? {} : styles.createMenuContainer}>
+                    <ArrowKeyFocusManager
+                        focusedIndex={focusedIndex}
+                        maxIndex={menuItemData.length - 1}
+                        onFocusedIndexChanged={(index) => setFocusedIndex(index)}
+                    >
+                        {_.map(menuItemData, (item, menuIndex) => (
+                            <MenuItem
+                                key={item.textTranslationKey}
+                                icon={item.icon}
+                                title={translate(item.textTranslationKey)}
+                                onPress={() => selectItem(item)}
+                                focused={focusedIndex === menuIndex}
+                            />
+                        ))}
+                    </ArrowKeyFocusManager>
+                </View>
+            </Popover>
+            {renderChildren()}
+        </>
+    );
 }
 
 AttachmentPicker.propTypes = propTypes;
```

Comment:
Add the `displayName` [field](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#function-component-style)